freqtrade/rpc/api_server/api_v1.py

```diff
@@ -88,6 +88,11 @@ def trades(limit: int = 0, rpc: RPC = Depends(get_rpc)):
     return rpc._rpc_trade_history(limit)
 
 
+@router.get('/trade/{tradeid}', response_model=OpenTradeSchema, tags=['info', 'trading'])
+def trade(tradeid: int = 0, rpc: RPC = Depends(get_rpc)):
+    return rpc._rpc_trade_status([tradeid])[0]
```

Comment:
uh - i think you will want some ... error handling here - assuming you get one trade might seem okish as you pass in one tradeID - but the return will be  a list with 1 or 0 trade - and in the "no trade found" case, it'll error out with a `KeyError`.

The difference between this and the below "delete_trade" is that the errorhandling  in the delete case is within the delete method - while here it isn't.